protocol/x/clob/types/keys.go

```diff
@@ -18,94 +18,91 @@ const (
 const (
 	// ProcessProposerMatchesEventsKey is the key to retrieve information about how to update
 	// memclob state based on the latest block.
-	ProcessProposerMatchesEventsKey = "process_proposer_matches_events"
+	ProcessProposerMatchesEventsKey = "ProposerEvents"
 
 	// SubaccountLiquidationInfoKeyPrefix is the prefix to retrieve the liquidation information
 	// for a subaccount within the last block.
-	SubaccountLiquidationInfoKeyPrefix = "subaccount_liquidation_info/"
+	SubaccountLiquidationInfoKeyPrefix = "SaLiqInfo/"
 
 	// LiquidationsConfigKey is the key to retrieve the liquidations config.
-	LiquidationsConfigKey = "liquidations_config"
+	LiquidationsConfigKey = "LiqCfg"
 
 	// EquityTierLimitConfigKey is the key to retrieve the equity tier limit configuration.
-	EquityTierLimitConfigKey = "equity_tier_limit_config"
+	EquityTierLimitConfigKey = "EqTierLimCfg"
 
 	// BlockRateLimitConfigKey is the key to retrieve the block rate limit configuration.
-	BlockRateLimitConfigKey = "block_rate_limit_config"
+	BlockRateLimitConfigKey = "RateLimCfg"
 
 	// ClobPairKeyPrefix is the prefix to retrieve all ClobPair
-	ClobPairKeyPrefix = "clob_pair/"
+	ClobPairKeyPrefix = "Clob/"
 )
 
 const (
 	// OrderAmountFilledKeyPrefix is the prefix to retrieve the fill amount for an order.
-	OrderAmountFilledKeyPrefix = "order_amount_filled/"
+	OrderAmountFilledKeyPrefix = "Filled/"
 
 	// BlockHeightToPotentiallyPrunableOrdersPrefix is the prefix to retrieve a list of potentially prunable orders
 	// by block height.
-	BlockHeightToPotentiallyPrunableOrdersPrefix = "block_height_to_potentially_prunable_orders/"
-
-	// OrdersFilledDuringLatestBlockKey is the key to retrieve the list of orders filled during the latest block.
-	OrdersFilledDuringLatestBlockKey = "orders_filled_during_latest_block"
+	BlockHeightToPotentiallyPrunableOrdersPrefix = "ExpHt/"
```

Comment:
Hmm any other ideas for this key name? Agree it's good to decrease length, but not sure `ExpHt` is descriptive

Maybe `PruneHt` instead?

Reply:
we don't need this to be descriptive right? this is only used for debugging. 

I think this is fine since we have `ExpTm`. let me know if you have a strong preference